docs/source/index.rst

```diff
@@ -6,6 +6,9 @@
 Welcome to gafferpy's documentation!
 ====================================
 
+`gafferpy <https://github.com/gchq/gafferpy>`_ is the Python shell for `Gaffer <https://github.com/gchq/Gaffer>`_. It enables creation and execution of operation chains in Python.
+This site is generated by `sphinx <https://www.sphinx-doc.org/en/master/>`_ from the gafferpy docstrings. For Gaffer documentation, instead go to the `Gaffer docs <https://gchq.github.io/gaffer-doc/latest/>`_.
```

Comment:
Could we add a short section explaining how to generate the docs HTML locally using this. Like [this section in gaffer-doc](https://github.com/gchq/gaffer-doc#building-the-documentation).